algebras/algebra/src/main/scala/endpoints4s/algebra/Urls.scala

```diff
@@ -108,6 +108,22 @@ trait Urls extends PartialInvariantFunctorSyntax {
       value: QueryStringParam[A]
   ): QueryString[A]
 
+  /** Builds a `QueryString` with one optional parameter, which has a default value.
+    *
+    * Examples:
+    *
+    * {{{
+    *   optQsWithDefault[Int]("page", 1) // optional `page` parameter, with default value 1
+    * }}}
+    *
+    * @param name Parameter’s name
+    * @tparam A Type of the value carried by the parameter
+    * @group operations
+    */
+  def optQsWithDefault[A](name: String, default: A, docs: Documentation = None)(implicit
+      value: QueryStringParam[Option[A]]
```

Comment:
Shouldn't this just be `value: QueryStringParam[A]`? In the body we can use

```scala
qs(name, docs)(optionalQueryStringParam(value)).xmap(_.getOrElse(default))(Option(_))
```

pretty much exactly as is done for optional fields.

Reply:
Great suggestion!
Fixed.